src/test/run-pass/yk_swt/collect-trace.rs

```diff
@@ -0,0 +1,28 @@
+// Copyright 2019 King's College London.
+// Created by the Software Development Team <http://soft-dev.org/>.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+#![feature(yk_swt)]
+
+use std::yk_swt::{start_tracing, stop_tracing};
+
+pub fn main() {
+    start_tracing();
+    let _ = work();
+    let trace = stop_tracing();
+    assert!(!trace.is_empty());
+}
+
+#[inline(never)]
+fn work() -> u64{
+    let mut res = 47;
```

Comment:
Any reason for the unusual numbers?

Reply:
Arbitrary.

Reply:
Can I suggest we choose less weird numbers? Otherwise it makes me think there's something specific about these (perhaps some threshold needs to be exceeded). I am a big fan of numbers < 10, or multiples of 10 ;)

Reply:
Can do, yeah.